shared/constants/types/tracker2.tsx

```diff
@@ -106,6 +105,7 @@ export type WebOfTrustEntry = {
 }
 
 export type State = {
+  readonly showTrackerSet: Set<string>
```

Comment:
when we load users the `usernameToDetails` changes (so this happens all the time), yet we figure out trackers (which basically never show) whenever this changes. instead pull this into its own map